net/base/BUILD.gn

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+static_library("utils") {
+  sources = [
+    "url_util.cc",
+    "url_util.h",
+  ]
+
+  deps = [
+    "//net",
+    "//url",
+  ]
+}
+
+source_set("unit_tests") {
+  testonly = true
+  sources = [ "url_util_unittest.cc" ]
+
+  deps = [
+    ":utils",
+    "//testing/gtest",
+    "//url",
```

Comment:
strictly speaking "//url" should be in public_deps of "utils" and don't appear explicitly here

Reply:
thanks, updated